lib/levels/forest.rb

```diff
@@ -45,27 +46,28 @@ def self.left(data)
   # If not, their journey continues. Deeper in to the forest they go.
   def self.right(data)
     message = [
-      'You start your journey along this road to see where it goes.',
-      'As you continue, the forest seems to get thicker around you.',
-      "Off to the side of the path, you see smoke rising. As you get closer you realise it's a house.",
-      'You come closer to the front of the house.'
+        "Your stomach rumbles, and it feels like you haven't eaten in a long time. You make your way towards the fire in the hope that there is something to eat. ",
+        'As you continue, the forest seems to thicken, and the trees seem to block the light, unnerving you as you walk further and further in',
+        "To your delight, however, the smoke seems to be getting closer. You move forward past a clearing and see a beautifully thatched old cottage",
+        'What do you do?'
     ]
     options = ['Walk past', 'Walk up and knock on the door']
     response = Gui.gui_message_intake(message, options)
-    if response == 'Walk past'
-      choice = 'go_past'
-    else
-      choice = 'knock'
-    end
+    choice = if response == 'Walk past'
+               'go_past'
+             else
+               'knock'
+             end
```

Comment:
Thinking about it, these could be refactored as ternary operatros, so like:
`choice  = response  == 'Walk past' ? 'go_past' : 'knock'`